### DISCUSSION TITLE
Clarity of field type/interface

### DISCUSSION THREAD

**Discussion by azrikahar** [directus/directus#9678] [Feature Requests]
In relation to https://github.com/directus/directus/issues/9645#issuecomment-964867232, this is a simple but rough preliminary iteration:

![S0COt2056G](https://user-images.githubusercontent.com/42867097/141086044-5a8446bf-9811-47f1-aaeb-633c9a615b57.gif)

Pros

- It's more verbose

Cons

- It's longer & may look cluttered

- `Datetime - dateTime` definitely looks very odd/redundant here, huge blocker for such implementation

Extra note: We can also probably run it through `formatTitle` to not show up as camelCase.

---

Or perhaps just adding `tz` is fine? Which was the original suggestion. But it might not be as clear to non-technical users to discern it as `timezone`.

**Comment by benhaynes** [comment]
Interesting approach, but the UI does get a bit too busy, and often is overflowing out of sight. I think the tooltip is enough, so...

* Remove the type from the field hover text (back to how it was)
* Clean up the tooltip for this additional information, see formatting below:
    * Run Types through title formatted

```
Name (String) — Input
```

**Comment by azrikahar** [comment]
This has been implemented in #9706
